allure-pytest/src/listener.py

```diff
@@ -128,10 +128,14 @@ def pytest_fixture_setup(self, fixturedef, request):
                                                stop=now(),
                                                status=get_outcome_status(outcome),
                                                statusDetails=get_outcome_status_details(outcome))
-
-        for index, finalizer in enumerate(fixturedef._finalizer or ()):
-            name = '{fixture}::{finalizer}'.format(fixture=fixturedef.argname, finalizer=finalizer.__name__)
-            fixturedef._finalizer[index] = allure_commons.fixture(finalizer, parent_uuid=container_uuid, name=name)
+        try:
```

Comment:
may be without code duplication and error handling? like this: 
...
finalizers = fixturedef._finalizer if hasattr(fixturedef, 'finalizer') else fixturedef._finalizers
for index, finalizer in enumerate(finalizers or ()):
...